static/js/stats/chart.js

```diff
@@ -229,7 +229,7 @@ const dayMsecs = 24 * 3600 * 1000;
     }
 
     // Transform xAxis based on time grouping (day, week, month) and range.
-    let pointInterval = (dayMsecs = 1 * 24 * 3600 * 1000);
+    let pointInterval = dayMsecs;
```

Comment:
Where is `1 * 24 * 3600 * 1000` now?

Reply:
`dayMsecs` is set to 24 * 36 * 1000 at the top of the file, so redefining it to that * 1 is redundant. `pointInterval` will further multiple the **constant** number of milliseconds in a day by whatever values it wants.